### ISSUE TITLE
html5 audio fails (no sound)

### ISSUE THREAD

**Issue by ingageco** [Telerik-Verified-Plugins/WKWebView#185]
first of all - this plugin is great! we have a simple HTML5 audio tag that plays a remote audio file - it was working in UIwebview (slow, but plays), but we can't get audio to work in WKWebView. Any suggestions?

**Comment by EddyVerbruggen** [comment]
Hi,

I added this to my index.html and it worked exactly the same for me as UIWebView:

```html
<audio src="http://www.tonycuffe.com/mp3/cairnomount_lo.mp3" preload="auto" controls></audio>
```

Do you have a samplefile which doesn't play perhaps?

**Comment by ingageco** [comment]
I've got multiple remote files - but let me try again tonight and report back.
